src/features/device-state/routes/state.ts

```diff
@@ -287,6 +330,29 @@ const stateQuery = _.once(() =>
 						},
 					},
 				},
+				should_be_managed_by__release: {
+					...releaseExpand,
+					$expand: {
+						...releaseExpand.$expand,
```

Comment:
I understand why you're doing this but there's a lot of overlap and overriding of attributes here and IMO this is prone to bugs. You primarily want to avoid writing the same thing each time but at the use site you also want to ensure that you can extend it safely without clobbering over attributes.

I believe converting `releaseExpand` to a function that returns a PineOptions (not sure about the exact type) and allows for parameterising `$select` and top-level `$expand` is much more readable and less error prone. You'd then use it like so:

```js
should_be_managed_by__release: {
  ...releaseExpand(['any', 'extra', '$select', 'attributes'], {
    belongs_to__application: { ... }
  })
}
```

Reply:
I refactored it into separate objects that can be composed as necessary.